source/disassemble.cpp

```diff
@@ -277,6 +281,7 @@ void Disassembler::EmitOperand(const spv_parsed_instruction_t& inst,
     case SPV_OPERAND_TYPE_IMAGE:
     case SPV_OPERAND_TYPE_MEMORY_ACCESS:
     case SPV_OPERAND_TYPE_SELECTION_CONTROL:
+    case SPV_OPERAND_TYPE_DEBUG_INFO_FLAGS:
```

Comment:
I think the whole list can be checked using `spvOperandIsConcreteMask()`?

Reply:
Yes.
It changes the switch to a sequence of if/else.  I didn't want to make that big of a change in this CL to add incremental functionality.  It should be a separate cleanup.